target/linux/ramips/dts/mt7628an_kroks_kndrt31r19.dts

```diff
@@ -0,0 +1,75 @@
+#include "mt7628an_kroks_1eth.dtsi"
+
+/ {
+	compatible = "kroks,kndrt31r19", "mediatek,mt7628an-soc";
+	model = "Kroks KNdRt31R19";
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+		serial1 = &uart1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status: status {
+			label = "kndrt31r19:led:status";
```

Comment:
model prefix should be dropped, just "led:status" etc.

Reply:
done (it was suggested I should write "done")